pkg/unikontainers/config.go

```diff
@@ -96,30 +99,32 @@ func getConfigFromSpec(spec *specs.Spec) (*UnikernelConfig, error) {
 	useDMBlock := spec.Annotations[annotUseDMBlock]
 
 	Log.WithFields(logrus.Fields{
-		"unikernelType":   unikernelType,
-		"unikernelCmd":    unikernelCmd,
-		"unikernelBinary": unikernelBinary,
-		"hypervisor":      hypervisor,
-		"initrd":          initrd,
-		"block":           block,
-		"blkMntPoint":     blkMntPoint,
-		"useDMBlock":      useDMBlock,
+		"unikernelType":    unikernelType,
+		"unikernelVersion": unikernelVersion,
+		"unikernelCmd":     unikernelCmd,
+		"unikernelBinary":  unikernelBinary,
+		"hypervisor":       hypervisor,
+		"initrd":           initrd,
+		"block":            block,
+		"blkMntPoint":      blkMntPoint,
+		"useDMBlock":       useDMBlock,
 	}).Info("urunc annotations")
 
 	// TODO: We need to use a better check to see if annotations were empty
-	conf := fmt.Sprintf("%s%s%s%s%s%s%s", unikernelType, unikernelCmd, unikernelBinary, hypervisor, initrd, block, blkMntPoint)
+	conf := fmt.Sprintf("%s%s%s%s%s%s%s%s", unikernelType, unikernelType, unikernelCmd, unikernelBinary, hypervisor, initrd, block, blkMntPoint)
```

Comment:
It seems there is a typo here. Instead of UnikernelType, it should be UnikernelVersion

Reply:
Yes, thanks for pointing it out. Fixed it.